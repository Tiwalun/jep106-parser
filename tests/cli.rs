use insta_cmd::{assert_cmd_snapshot, get_cargo_bin};
use std::process::Command;

#[test]
fn help_output() {
    assert_cmd_snapshot!(Command::new(get_cargo_bin("jep106-build")).arg("-h"));
}
