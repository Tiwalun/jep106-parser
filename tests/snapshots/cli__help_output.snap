---
source: tests/cli.rs
info:
  program: jep106-build
  args:
    - "-h"
---
success: true
exit_code: 0
----- stdout -----
jep106-build 0.1.0

USAGE:
    jep106-build --pdf <PDF> --jep_version <VERSION>

OPTIONS:
    -h, --help                     Print help information
        --jep_version <VERSION>    
        --pdf <PDF>                

----- stderr -----

